sloyka/src/utils/data_processing/city_services_extract.py

```diff
@@ -5,6 +5,7 @@
 from rapidfuzz import fuzz
 from typing import List
 from sloyka.src.utils.constants import CITY_SERVICES_NAMES
+from sloyka.src.utils.data_preprocessing.preprocessor import PreprocessorInput
 
 tagger = SequenceTagger.load("Glebosol/city_services")

```

Comment:
I think we could move such things into init func to fasten main import of sloyka and its modules